storage/storageUnit/storageunit.go

```
@@ -112,6 +115,14 @@ func (s *Unit) Put(key, data []byte) error {
 	return err
 }
 
+// Close will close unit
+func (s *Unit) Close() error {
+	err := s.persister.Close()
```

The Close should return an error but you log the error and always return nil, you should return the error instead.

---

done.

---

:+1: